pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.poetry]
+name = "exasol-transformers-extension"
+version = "0.1.0"
+description = "An Exasol extension to use state-of-the-art pretrained machine learning models via the transformers api."
+
+authors = [
+    	"Umit Buyuksahin <umit.buyuksahin@exasol.com>",
+    	"Torsten Kilias <torsten.kilias@exasol.com>"
+]
+
+readme = 'README.md'
+
+repository = "https://github.com/exasol/transformers-extension"
+homepage = "https://github.com/exasol/transformers-extension"
+
+
+keywords = ['exasol']
+
+[tool.poetry.dependencies]
+python = ">=3.8,<4.0"
+Jinja2 = "^3.0.3"
+importlib-resources = "^5.4.0"
+exasol-bucketfs-utils-python = { git = "https://github.com/exasol/bucketfs-utils-python.git", branch = "main" }
```

Comment:
Is it possible to use fixed releases here instead of the main branch?

Reply:
not yet, the target repo is still quite in flow, bu itt is planned

Reply:
yes it is possible, if you state the tag info rather than branch info.
Since some updates are not released immediately, we get them from the main branch.